server/channels/api4/user.go

```diff
@@ -1335,6 +1335,12 @@ func patchUser(c *Context, w http.ResponseWriter, r *http.Request) {
 		c.SetInvalidParam("user_id")
 		return
 	}
+	if ouser.IsBot {
```

Comment:
Should this maybe come after the `AddEventPriorState` below? That seems to fit with how the original user is included in the audit if one admin attempts to modify another